### PR TITLE
docs-user-Getting Started adjustments

### PR DESCRIPTION
**Pull request by LiborBatek**
## Brief description
Small typo fixes here and there, additional info on install/ running OP.

### CODE REVIEW

File: website/docs/artist_getting_started.md [hunk 1]
```diff
@@ -18,13 +18,17 @@ If this is not the case, please contact your administrator to consult on how to
...
 
-**openpype_gui.exe** is the most common for artists. It runs OpenPype GUI in system tray. From there you can run all the available tools. To use any of the features, OpenPype must be running in the tray.
+first most common one by using OP icon on the Desktop triggering
 
-**openpype_console.exe** in useful for debugging and error reporting. It opens console window where all the necessary information will appear during user's work. 
+**openpype_gui.exe** suitable for artists. It runs OpenPype GUI in the OS tray. From there you can run all the available tools. To use any of the features, OpenPype must be running in the tray.
+
+or alternatively by using
+
+**openpype_console.exe** located in the OpenPype folder which is suitable for TDs/admins for debugging and error reporting. It opens console window where all the necessary information will appear during user's work.
```
Comment (BigRoy) [inline website/docs/artist_getting_started.md:31]: Aren't both executables located in that folder? I'd either mention it prior to both or not mention it only here. This makes it feel like `openpype_gui.exe` is NOT located there and leaves me wondering where `openpype_gui.exe` would be located.

File: website/docs/artist_getting_started.md [hunk 2]
```diff
@@ -18,13 +18,17 @@ If this is not the case, please contact your administrator to consult on how to
...
 admin, supervisor or production, because OpenPype versions and executables might not be compatible between studios.  
 
-To install OpenPype you just need to unzip it anywhere on the disk
+Installing OpenPype is possible by Windows installer or by unzipping it anywhere on the disk from downloaded ZIP archive.
 
-To use it, you have two options
+There are two options running OpenPype
 
-**openpype_gui.exe** is the most common for artists. It runs OpenPype GUI in system tray. From there you can run all the available tools. To use any of the features, OpenPype must be running in the tray.
+first most common one by using OP icon on the Desktop triggering
```
Comment (BigRoy) [inline website/docs/artist_getting_started.md:25]: Admittedly this is hard for me to follow...

I have two options to install:
- Installer
- Unzipping ZIP archive

I have two options to run, but the added  'first most common one' mention makes it hard to follow because it's not visually connected to the line that it should be connected to.
First off, I think when unzipping the ZIP archive, there won't be an OP icon on the desktop.
Second, I'd refrain from abbreviating OpenPype to OP early on in the getting started guide, just to remain very explicit to the end user.

Comment (LiborBatek) [reply website/docs/artist_getting_started.md]: @BigRoy Yeah, you are right, good points! Sry just starting with writing docs being noobish! Thanks for your input will take it on my mind when correcting it!

**Comment by LiborBatek** [comment]
Did make propery changes according to @BigRoy comments. Should be fine now.

File: website/docs/artist_getting_started.md [hunk 3]
```diff
@@ -15,70 +16,60 @@ If this is not the case, please contact your administrator to consult on how to
...
 
-</TabItem>
-</Tabs>
+When you first start OpenPype, you will be asked to fill in some basic informations.
 
+### MongoDB
 
-## First Launch
+In most cases that will only be your studio MongoDB Address.
+It's a URL that you should have received from your Studio admin and most often will look like this
```
Comment (BigRoy) [inline website/docs/artist_getting_started.md:48]:
```suggestion
In most cases you will only have to supply the MongoDB Address.
It's the database URL you should have received from your Studio admin and often will look like this 
```

File: website/docs/artist_getting_started.md [hunk 4]
```diff
@@ -15,70 +16,60 @@ If this is not the case, please contact your administrator to consult on how to
...
+ `mongodb://192.168.100.15:27071`
 
-It is a URL that you should receive from you studio and most often will look like this `mongodb://username:passwword@mongo.mystudiodomain.com:12345` or  `mongodb://192.168.100.15:27071`, it really depends on your studio setup. When OpenPype Igniter
+it really depends on your studio setup. When OpenPype Igniter
 asks for it, just put it in the corresponding text field and press `install` button.
 
 ### OpenPype Version Repository
 
-Sometimes your studio might also ask you to fill in the path to it's version
+Sometimes your Studio might also ask you to fill in the path to it's version
```
Comment (BigRoy) [inline website/docs/artist_getting_started.md:61]:
```suggestion
Sometimes your Studio might also ask you to fill in the path to its version
```

File: website/docs/artist_getting_started.md [hunk 5]
```diff
@@ -15,70 +16,60 @@ If this is not the case, please contact your administrator to consult on how to
...
 repository. This is a location where OpenPype will be looking for when checking
 if it's up to date and where updates are installed from automatically.
```
Comment (BigRoy) [inline website/docs/artist_getting_started.md:63]:
```suggestion
repository. This is a location where OpenPype will search for the latest versions, check
if it's up to date and where updates are installed from automatically. 
```

File: website/docs/artist_getting_started.md [hunk 6]
```diff
@@ -15,70 +16,60 @@ If this is not the case, please contact your administrator to consult on how to
...
 if it's up to date and where updates are installed from automatically. 
 
-This pat is usually taken from the database directly, so you shouldn't need it. 
+This path is usually taken from the database directly, so you shouldn't need it. 
 
 
 ## Updates
 
-If you're connected to your studio, OpenPype will check for, and install updates automatically every time you run it. That's why during the first start, it will go through a quick update installation process, even though you might have just installed it. 
+If you're connected to your Studio, OpenPype will check for, and install updates automatically every time you run it. That's why during the first start, it will go through a quick update installation process, even though you might have just installed it.
```
Comment (BigRoy) [inline website/docs/artist_getting_started.md:70]:
```suggestion
If you're connected to your Studio, OpenPype will check for, and install updates automatically every time you run it. That's why during the first start it can go through a quick update installation process, even though you might have just installed it. 
```

File: website/docs/artist_getting_started.md [hunk 7]
```diff
@@ -15,70 +16,60 @@ If this is not the case, please contact your administrator to consult on how to
...
+Installing OpenPype is possible by Installer or by unzipping downloaded ZIP archive to any drive location.
 
-To use it, you have two options
+> For more detailed info about installing OpenPype please visit [Installation section](artist_install.md).
 
-**openpype_gui.exe** is the most common for artists. It runs OpenPype GUI in system tray. From there you can run all the available tools. To use any of the features, OpenPype must be running in the tray.
+---
 
-**openpype_console.exe** in useful for debugging and error reporting. It opens console window where all the necessary information will appear during user's work. 
+You can run OpenPype by desktop "OP" icon (if exists after installing) or by directly executing
```
Comment (BigRoy) [inline website/docs/artist_getting_started.md:28]:
```suggestion
You can run OpenPype by desktop "OP" icon (if it exists after installing) or by directly executing
```

**Review by jakubjezek001 — APPROVED** [review]

**Review by BigRoy — APPROVED** [review]
Looks good to me - I still think we can improve [this](https://github.com/ynput/OpenPype/pull/4365#discussion_r1091724848) situation slightly by having that information either merged here or linking back/forth. Having the screenshots is actually really nice and would be good to maybe move here too?

**Comment by LiborBatek** [comment]
@BigRoy Good Idea...I have just simply linked the pic from install section of the systray icon and slightly adjusted Note section above...
Thx for all your reviews and ideas!
